src/_guides/libraries/library-tour.md

```diff
@@ -961,9 +966,13 @@ void main() {
 #### Iteration
 
 The [Iterable][] and [Iterator][] classes
-support for-in loops. Extend (if possible) or implement Iterable
-whenever you create a class that can provide Iterators for use in for-in
-loops. Implement Iterator to define the actual iteration ability.
+support sequential access to a collection of values.
+To practice using these collections,
+follow the [Iterable collections codelab](/codelabs/iterables).
+
+If you create a class that can provide Iterators for use in for-in loops,
```

Comment:
To be consistent with elsewhere, since `Iterable` is the type, should we not capitalize `Iterators`? Also, in a previous tip, you put `for-in` as inline code. Do you want to do so here as well?

```suggestion
If you create a class that can provide iterators for use in `for-in` loops,
```

Reply:
I'm glad you're thinking about this. :) I was _so_ tempted to change the capitalization/font, but I decided to make minimal changes here, for consistency. However, I might soon undertake a wholesale update of the language & library tours to use code font. (I hate the look, but it's helpful for auto-translation and I suppose clarity.)

The relevant issue is #2269.